tf2_kdl/test/test_tf2_kdl.cpp

```diff
@@ -33,16 +33,16 @@
 #include <tf2_kdl/tf2_kdl.h>
 #include <kdl/frames_io.hpp>
 #include <gtest/gtest.h>
+#include <rclcpp/rclcpp.hpp>
 #include "tf2_ros/buffer.h"
+#include <tf2/convert.h>
 
-
-tf2_ros::Buffer* tf_buffer;
+std::unique_ptr<tf2_ros::Buffer> tf_buffer;
 static const double EPS = 1e-3;
 
 TEST(TfKDL, Frame)
 {
-  tf2::Stamped<KDL::Frame> v1(KDL::Frame(KDL::Rotation::RPY(M_PI, 0, 0), KDL::Vector(1,2,3)), builtin_interfaces::msg::Time(2.0), "A");
-
+  tf2::Stamped<KDL::Frame> v1(KDL::Frame(KDL::Rotation::RPY(M_PI, 0, 0), KDL::Vector(1,2,3)), tf2::TimePoint(tf2::durationFromSec(2.0)), "A");
```

Comment:
The windows M_PI warning can be worked around with an extra #define 

See https://github.com/ros2/geometry2/commit/dc7b000 for an example

Reply:
This should be resolved now (I'll run CI again to ensure it).